app/src/main/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeSheet.kt

```diff
@@ -24,7 +24,6 @@ class ConfirmFacilityChangeSheet : BottomSheetActivity(), ConfirmFacilityChangeU
     lateinit var component: ConfirmFacilityChangeComponent
```

Comment:
There is a typo in the commit message
"Close facility selection when **cancel** is clicked"